almalinux-deploy.sh

```diff
@@ -38,6 +38,8 @@ REMOVE_PKGS=("centos-linux-release" "centos-gpg-keys" "centos-linux-repos" \
                 "rocky-release" "rocky-gpg-keys" "rocky-repos" \
                 "rocky-obsolete-packages")
 
+module_list_enabled=""
+module_list_insalled=""
```

Comment:
typo. should be **installed**

Reply:
isn't any typo. enabled/installed modules in system now.

Reply:
you've missed "t" in module_list_insalled